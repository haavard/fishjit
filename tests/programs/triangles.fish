f99+0{:}1=?.~~"  "oo:2,:1%-v
 -1  oooo  "____"  v!?  )0:/!
 " /"oa~~.?=1}:{24~/:oo
 v!?)0:-1o"\"v!?)0:/!-1ooo"  /"
 /v   ~o"/"oa/!
 !\:0)?!;"\__"ooo1-:0)?!;"/"o1-

http://codegolf.stackexchange.com/a/48188
