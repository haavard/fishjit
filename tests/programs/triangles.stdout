  ____________________________
 /\  /\  /\  /\  /\  /\  /\  /\
/__\/__\/__\/__\/__\/__\/__\/__\